  <h1>Elizabeth H. Blackburn - Biographical</h1>
<p><img src="blackburn.jpg" class="bio-img">
<b>Childhood<br>
</b>I was born in the small city of Hobart in Tasmania, Australia, in 1948. My
  parents were family physicians. My grandfather and great grandfather on
  my mother's side were geologists. My great-grandfather on my father's side,
  before coming to Australia as a minister of the Church of England, had lived
  for some time in Hawaii, where he had collected Coleoptera (beetles). He
  continued his collecting in Australia, eventually selling his collection to the
  British Museum of Natural History. My uncle and aunt (my father's sister
  and my mother's brother) were also both family physicians, who moved to
  England, married there and permanently settled there to practice medicine
  and raise their families.</p>
<p>I was the second child of eventually seven siblings. I spent my first 4
  years living in the tiny town of Snug, by the sea near Hobart. Curious about
  animals, I would pick up ants in our backyard and jellyfish on the beach.
  Then my family moved to Launceston, a town in northern Tasmania. Our
  first house, at 120 Abbott Street, was a one-storied, verandahed house of
  typical Australian suburban architecture. I started kindergarten at a girls'
  school, Broadland House Girls Grammar School in Launceston (Figure 1).</p>
<table width="252" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_1.jpg" width="252" height="427" alt="Elizabeth Blackburn and her
sister Katherine"></td>
</tr>
<tr>
<td class="caption"><i>Figure 1</i>. Elizabeth Blackburn (right) and her
sister Katherine ready for Elizabeth's first day at
school in Launceston, Tasmania. Circa 1953.</td>
</tr>
</table>
 <p>&nbsp;</p>
 <p>I kept tadpoles in rapidly-smelly-becoming glass jars in a back living room
   at home. When I was a preteen we moved to a larger house called Elphin
   House, which had a good-sized garden (Figure 2). Over the years we had
   many pets: at one stage I enumerated the family menagerie of the moment
   as consisting of budgerigars and canaries in an aviary in one corner of the
   garden, goldfish in a garden pond, chickens and pullets (for eggs and the occasional
   roast fowl) in a hen coop and henhouse, rabbits and guinea pigs in
   cages, and cats and a dog, who lived all over the house and garden. I was fond
  of all these animals, and of animals and nature in general.</p>
  
  <img src="blackburn_autobio_2.jpg" alt="All seven Blackburn family siblings">
<p class="photo-gallery-caption"><i>Figure 2</i>. All seven Blackburn family siblings in the garden at their home, 3 Olive Street, Launceston, Tasmania. From left to right, back row: Andrew, Elizabeth, Katherine, John, Barbara; front row: Caroline, Margaret. Circa 1965.</p>



<p>&nbsp;</p>
<p>Perhaps arising from a fascination with animals, biology seemed the most
  interesting of sciences to me as a child. I was captivated by both the visual
  impact of science through science books written for young people, and
  an idea of the romance and nobility of the scientific quest. This latter was
  especially engendered by the biography of Marie Curie, written by her daughter,
  which I read and reread as a child. By the time I was in my late teens it
  was clear to me that I wanted to do science. I was educated at Broadland
  House Girls Grammar School, and received a generally excellent education.
  However, physics was not offered, so I took physics classes offered in the
  evenings at the local public high school. Latin and Greek were not taught
  at my girls' school either, a gap in my education that I rather regret later in
  life. But my school did provide an excellent piano teacher, Helen Roxburgh,
  by whom I was taught all throughout my school years in Launceston. I loved
  playing the piano, and even at one time wistfully hoped that I might become
  a musician. Fortunately I was also quite realistic about this, because I
  recognized that I was competent rather than greatly talented at piano playing,
  so I went in the direction of science.</p>
<p>My family moved, after some family disruptions, to the city of Melbourne,
  Australia, in time for me to complete my last year of high school at University
  High School. There I gained the confidence that I needed to apply for the
  undergraduate science degree at the University of Melbourne.</p>
<p>
  <b>University Education</b><br>
  I chose biochemistry as my major and graduated after 4 years with an
  Honours degree in Biochemistry. During that time, I had come to love
  biochemistry research, although I was just getting my feet wet in laboratory
  research.</p>
<p>
  The Chair of the Biochemistry Department, Frank Hird, then offered
  me a position as a Master's student in his research laboratory, where they
  investigated the biochemistry of amino acid metabolism. My undergraduate
  Honours thesis research advisors, Theo Dopheide and the late Barrie
  Davidson, had advised and encouraged me to do my Ph.D. abroad. Barrie
  in particular had urged me to consider going to the MRC Laboratory of
  Molecular Biology (LMB) in Cambridge, England, where he had done postdoctoral
  research. But in order to be accepted as a Cambridge Ph.D. student
  in biology, those from outside Britain were required to have done a year of
  research. The Master's degree with Frank Hird, studying the metabolism of
  glutamine in the rat liver, would constitute this required year.
  Frank Hird taught his laboratory group members the joy and aesthetics of
  research. He said he thought each experiment should have the beauty and
  simplicity of a Mozart sonata. His laboratory group, dominated by his strong
  personality, was cohesive and we would sometimes drive to the hilly areas outside
  Melbourne, all piled into his car, Mozart playing loudly on the car radio,
  to have an outdoor lunch picnic among trees and wildflowers.
  While I was still in Frank Hird's lab, <a href="/nobel_prizes/chemistry/laureates/1958/">Fred Sanger</a> visited Melbourne.
  Frank Hird had done research in England on amino acids with Fred Sanger
  shortly after the Second World War, providing an introduction to Fred in
  which Frank encouraged me to tell Fred of my hope to study for my Ph.D.
  in Cambridge at the LMB. It was arranged that I would join Fred's lab in the
  LMB and I was admitted as a Cambridge Ph.D. student.</p>
<p>The adventure of setting off to England, away from home and family, was a
  huge step, but I felt ready. My aunt and uncle and their family in Cambridge,
  who lived close to the LMB, became my anchor of a family away from home.
  I loved the LMB, the science being done there, the atmosphere of being at
  the epicenter of molecular biology, the intensity of the scientists and the
  constant discussions about science. It was a world of complete immersion.
  For my Ph.D. thesis research, I carried out sequencing of regions of bacteriophage
  phiX 174, a small single stranded DNA bacteriophage. I transcribed
  fragments of the phage DNA into RNA and then used the methods that
  Fred had pioneered for piecing together RNA sequences. We combined the
  sequences derived by this method with the DNA sequencing that had been
  done by John Sedat and Ed Ziff and Francis Galibert, members of Fred's lab. All the sequences jibed. The first sequence of a 48 nucleotide fragment
  of this tiny bacteriophage DNA genome was a great excitement. I took it to
  show to my mathematically-talented Cambridge cousin, who was then about
  12 years old, to see if any patterns emerged to his mathematically-inclined
  eye. He pointed out the repeats, but it was premature to think of analyzing
  DNA sequence patterns!</p>
<p><b>To the United States</b><br>
  The world of discovering DNA sequences was opening up and I was entranced
  by its possibilities. I had planned to do a postdoctoral fellowship, beginning
  in 1975, with Howard Goodman and his close associate Herb Boyer
  of UCSF, a mutual decision made after an interview&ndash;cum-conversation Herb
  and I had walking in the garden of a monastery in Belgium at which we were
  attending a scientific conference, I still as a graduate student. But then love
  intervened: John Sedat and I decided to marry, and as John was going to Yale,
  I decided to see if I could change my postdoctoral research plans (for which
  I had obtained an Anna Fuller Fellowship to work at UCSF) to a laboratory at
  Yale. Howard Goodman wrote me a kind and understanding letter upon my
  letting him know the reasons for my change of plan, and I began inquiries
  into possibilities of a laboratory for my postdoctoral training at Yale.</p>
<p>
  Thus it was that love brought me to a most fortunate and influential
  choice: Joe Gall's lab at Yale. After a few hiccups engendered by misplaced
  international mail and other factors, at the beginning of 1975 Joe accepted
  me as a postdoctoral fellow in his lab, to which I was allowed to transfer my
  Anna Fuller Fellowship. I immediately began to work on finding ways to
  accomplish the sequencing of the DNA found at the terminal regions of the
  abundant, short, linear ribosomal gene-carrying &quot;minichromosomes&quot; that
  Joe and his colleagues, in parallel with Jan Engberg of Denmark, had discovered
  in the somatic nucleus of the ciliated protozoan <i>Tetrahymena</i> thermophila
  (which was at that time called <i>Tetrahymena pyriformis</i>, shortly thereafter to be
  renamed <i>Tetrahymena thermophila</i>).</p>
<p><b>To the University of California</b><br>
  After finishing my postdoctoral training in Joe Gall's lab at the end of
  1977, John Sedat and I, having married in 1975, moved to San Francisco,
  California. There John had accepted a position as Assistant Professor at the
  University of California San Francisco (UCSF). I had applied for several
  positions as an Assistant Professor in a variety of Universities and had been
  rejected from many of them, a discouraging experience. I had applied for
  such a position in the Department of Molecular Biology at the University of
  California Berkeley, but had not yet heard whether I was in the running for
  it. In the meantime, UCSF offered me a research track position and space in
  the Department of Biochemistry in the Genetics unit headed by Herb Boyer.
  My first NIH grant was the source of funding for my salary and research
  expenses. I had written this grant application with the encouragement of
  UCSF, in order to pursue my research on <i>Tetrahymena</i> telomeres and their
  associated proteins. This work grew out of that I had done in Joe Gall's
  laboratory at Yale. My grant was funded by the NIH General Medicine
  Institute. Unsure of my chances at obtaining funding, I had sent the same
  grant application to the National Institutes of Health, the National Science
  Foundation and the American Cancer Society, hoping for funding from
  any one of these. Reflecting the more informal scientific habits of the basic
  sciences community in those days, some time later one of the grant reviewers
  told me that he had been so intrigued by my photograph of the autoradiogram,
  showing that telomeric DNA in <i>Tetrahymena</i> was mysteriously packaged
  as something other than nucleosomes, that he had kept the photograph.</p>
<p>Then UC Berkeley offered me an associate professor position in the
  Molecular Biology Department, which I immediately accepted. Once again,
  I transferred my funding from UCSF, this time to my own laboratory, at UC
  Berkeley.</p>
<p>
  Because research was, and still is, such a central part of my life, my autobiography
  would be incomplete without describing my research experiences.
  Thus, to convey a fuller flavor of them, here I describe some of the events
  of my early scientific research on the molecular nature of ends of chromosomes.</p>
<p><b>Early Work on the DNA at the Ends of Eukaryotic Chromosomes</b><br>
  Very soon after arriving in Joe Gall's laboratory at Yale in early 1975, I started
  to apply methods for obtaining terminal DNA sequences to the <i>Tetrahymena</i> rDNA molecules. I had learned a collection of methods in Fred Sanger's
  laboratory in Cambridge, England, where I had just completed my Ph.
  D. Eager to sequence the end regions of these minichromosomes, with
  Joe's encouragement I set out right away early in 1975 to use end-labeling
  techniques on them. I incorporated <sup>32</sup>P isotope-radiolabeled deoxynucleosides
  residues into the <i>Tetrahymena</i> rDNA molecules using commercially
  available DNA polymerases for <i>in vitro</i> DNA repair enzymatic reactions.
  The results were immediately promising. First, it became clear that the end
  regions of the rDNA were being selectively labeled by certain combinations
  of <sup>32</sup>P isotope radiolabeled nucleoside triphosphate substrates. And by June
  1975, I had become tremendously excited: I had obtained my first autoradiogram
  of the two-dimensional separation of the <sup>32</sup>P labeled depurination
  products. A strong signal of a run of 4 cytosine (C) residues was apparent.
  Furthermore, each such C4 sequence was flanked by a purine residue (that
  is, an adenosine (A) or a guanosine (G) residue; this initial data did not show
  which). The way the depurination reaction worked was the following: Ken
  Burton, a New Zealander, had shown that a chemical reaction could be done
  that cleaves the DNA backbone on both sides of every purine nucleotide but
  leaves intact any runs of pyrimidine nucleotides (such as C residues) that are
  uninterrupted by purine nucleotides. This so-called depurination method,
  when applied to a complex sequence DNA like a whole bacteriophage
  genome, was further made useful by Vic Ling, when he was a postdoctoral
  fellow in Fred Sanger's lab in Cambridge, England. Vic had shown that
  the resulting short pyrimidine tracts (mono-, di- tri-nucleotides, etc) would
  yield a pattern of products like a grid when a 2D fractionation method was
  used (see <a href="blackburn-lecture.html">2009 Nobel Lecture by Elizabeth Blackburn</a>, in this volume).
  The most frequent products, on a random basis, of course are mono and
  dinucleotides, with the longer tri- and tetranucleotide tracts being less and
  less abundant on a random basis. Thus this strong C4 spot was interesting
  and informative.</p>
<p>It was also clear to me that the rDNA molecules were not simply lambda
  phage-like DNA. First, the terminal fragments were heterogeneous. In
  contrast, for any one type of lambdoid phage, in every viral particle the DNA
  molecule is just the same &minus; a perfect carbon copy &minus; as in every other viral
  particle. Second, on a per DNA molecule basis, much more incorporation
  of radiolabeled nucleotide precursor substrates occurred than expected if
  rDNA molecule ends were like those of the sticky DNA ends of lambda
  phages. It was also fortunate that the C4 repeat had such a regular nature
  and that it happened to have four Cs in a row. Even three C's would have
  been striking, but harder to unravel. As it was, the C4 spot consistently stood
  out like a beacon through my repeated productions of 2D fractionations
  (&quot;homochromograms&quot; as we called them in Fred Sanger's lab).</p>
<p>
  Next, I needed to validate independently that what I was radiolabeling
    <i>in vitro</i> validly reflected the rDNA sequence, and also try to get a closer
  estimate of the number of C4 runs per rDNA molecule. I therefore decided
  I needed to <sup>32</sup>P isotope-label the rDNA <i>in vivo</i>. The vast majority of the <sup>32</sup>P
  isotope (chemically in the form of inorganic phosphate ions) taken into
  the cells would be incorporated into other molecules, including the much
  more abundant cellular RNA, with very little ending up in DNA, and even
  less in the rDNA (in <i>Tetrahymena</i>, only a percent or so at most of the DNA is
  rDNA). Therefore, with some trepidation I asked Joe Gall for permission to
  order sufficient <sup>32</sup>P phosphate to be added to the cell growth medium for
  labeling the rDNA. This meant handling 2 milliCuries at once, which Joe's
  lab had not done before. But I knew that this was the only way currently available
  to get a sufficient amount of <sup>32</sup>P into rDNA to detect the C4 spot in an
  autoradiogram, above background. Possibly with some trepidation on his
  side too, Joe agreed. I worked in the &quot;hot room&quot;, a room set aside for doing
  work with radioactive isotopes. On October 9, 1975 the <sup>32</sup>P was shipped to
  Yale. My laboratory notebook from that time reads: &quot;<sup>32</sup>P stored in refrig. until
  use &ndash; assayed for 10/14 . 3 pm 10/14. Zeroed [the cell culture] on 1% PPS
  medium blank ... Added 2 mCi <sup>32</sup>P as h3 PO4 in water in 1 ml from syringe.&quot;
  For the next preparation I raised it to 5 milliCuries. Curious about this very
  radioactive departure from the more usual activities of the lab, my Gall
  labmates periodically looked in through the window set in the door of the
  hot room as I worked.</p>
<p>By October 22, 1975, I had the <sup>32</sup>P rDNA purified. I triumphantly wrote
  down in my notebook that day my plans for this precious sample:</p>
<p>
  &quot;1) Depurination<br>
  2) denaturing gel after EcoRi treatment<br>
  3) 1.4% agarose gel&quot;</p>
<p>
  One by one, I inflicted various nucleases on the terminal region of the
  rDNA. I found it could be selectively radiolabeled using one triphosphate <sup>32</sup>P
  labeled at a time. Then I carried out the battery of analyses possible at the
  time that would allow me to piece together the nucleotide sequence: I
  digested the radiolabeled end regions with Endonuclease IV nuclease or
  micrococcal nuclease, and performed depyrimidations and nucleotide nearestneighbor
  analyses, and spleen phosphodiesteriase digestions (Figure 3).</p>
  
<table width="350" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_3.jpg" width="350" height="457" alt="Lab notes"></td>
</tr>
<tr>
<td class="caption"><i>Figure 3</i>. Lab Notes 02/12/1976.</td>
</tr>
</table>
<p>&nbsp;</p>  
<p>Piecing together the rDNA end sequence was a matter of careful puzzle-solving.
  At an intermediate point, my notebooks of the time show that I had to consider
  two possibilities &ndash; CCCCAG and CCCCAA repeats (Figure 4). But it was
  apparent that there were a large number of repeated copies.</p>
  
<table width="350" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_4.jpg" width="350" height="453" alt="Lab notes"></td>
</tr>
<tr>
<td class="caption"><i>Figure 4</i>. Lab Notes.</td>
</tr>
</table>
<p>&nbsp;</p>
<p>But by April 8, 1976, I was confident that the correct sequence was deduced,
  because the entry in my laboratory note-book page headed with that date
  reads:</p>
<table width="470" border="0" cellspacing="2" cellpadding="2" summary="Sequence data">

  <tr>
    <td colspan="4"><u>&quot;Sequence data from:</u></td>
  </tr>
  <tr>
    <td colspan="4">1) Depurination</td>
  </tr>
  <tr>
    <td width="169">2) Endo IV</td>
    <td colspan="3">&ndash; deps [my abbreviation for depurinations]</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td colspan="3">&ndash; nearest neighbours</td>
  </tr>
  <tr>
    <td>3) partial micrococcal</td>
    <td colspan="3">&nbsp;</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td width="50">&nbsp;</td>
    <td width="305" colspan="2">&ndash; deps</td>
  </tr>
  <tr>
    <td>&nbsp;  </td>
    <td colspan="3">&ndash; nearest neighbours</td>
  </tr>
  <tr>
    <td colspan="4">partial spdes [my abbreviation for spleen phosphodiesterase digestions]</td>
  </tr>
  <tr>
    <td colspan="4">4) Gel analysis of Endo IV digest of restriction fragments t ends of rDNA</td>
  </tr>
  <tr>
 <td>&nbsp;</td>
    <td colspan="3">&#8594; AACCCC repeated</td>
  </tr>
  <tr>
    <td>&nbsp;</td>
    <td colspan="3">&#8594; AACCCCAACCCCAACCCC) etc.&quot;</td>
  </tr>
</table>

<p>&nbsp;</p>
<p>Another laboratory note-book page, dated August 17, 1976, shows I was
  already referring, in a routine way, to (CCCCAA)<sub>n</sub> sequence &ndash; by then in
  the course of experiments designed to see whether this same repeated
  sequence was also present in the other (much longer) chromosomal DNAs
  of <em>Tetrahymena.</em></p>
<p>I put together a picture which tried to take into account all of my many observations.
  The deduced sequence consisted of a tandem array of CCCCAA
  repeats. One experiment done in 1979, radiolabeling the rDNA using just
  <sup>32</sup>P-labeled dCTP, and unlabeled dAT P, and separating the products on a
  denaturing gel electrophoresis, showed this visually as a beautiful ladder of
  tiger stripes extending up the gel. The size of every band in this regular
  ladder was 6 bases more than the band below it! This strikingly characteristic
  visible pattern of bands presaged the pattern that would later become
  important for our discovery of telomerase enzyme activity, as described in
  this volume in the <a href="/nobel_prizes/medicine/laureates/2009/greider-lecture.html">2009 Nobel Lecture by my co-awardee, Carol Greider</a>.</p>
<p>In the months following April 1976, much of my effort was also devoted to
  trying to understand the arrangement of the strand discontinuities along the
  tract of CCCAA repeat DNA. For this, I performed a great many experiments
  following the kinetics and specificity of radiolabeling the RNA end regions,
  using multiple different enzymes and protocols and analyses. This also was
  a matter of piecing things together &ndash; there was no template for me to work
  from as this was all uncharted territory.</p>
<p>In the late 1970s and early 1980s, I did a variety of radiolabeling experiments
  trying to divine the structure right at the termini of telomeres of both
  ciliated protozoans and yeast linear plasmids. I could put together a composite
  but still incomplete picture. Some of the features, I realize in retrospect,
  might be attributed to the terminal G strand sequence at the very ends of the
  telomeres assuming G-G paired or G-quartet structures. But other features
  are not so readily explainable. Why I was able to label the strands with DNA
  polymerase or a kinase to get the patterns of labeled strands and nucleotides
  I did has still not been completely fitted into a coherent view of the molecular
  structure of DNA ends. Various kinds of <i>in vitro</i> radio-labeling experiments
  had suggested that, in both <i>Tetrahymena</i> rDNA and the macronuclear DNAs
  of hypotrichous ciliates, there is a short overhang of the G-rich strand consisting
  of a few telomeric repeat sequences. Currently the view is that in mammalian
  telomeres there is a long protruding G-rich strand. Yet this does not
  take into account the clear evidence for the short C strand repeat oligonucleotides
  that I discovered can be readily melted off the telomeric DNA. This I
  found for both the <i>Tetrahymena</i> rDNA minichromosome molecules and linear
  plasmids purified from yeast. These tiny telomeric sequence oligonucleotides
  could be radiolabeled and clearly identified by two dimensional fractionations.
  However their significance is still unknown. To this day, aspects of the
  structure at the very terminal region of the telomeric DNA are enigmatic; the
  very ends of chromosomes remain as challenges.</p>
<p><b>Telomere Proteins: Earliest Attempts and Failures</b><br>
  In my early work, my molecular views of telomeres were first focused on the
  DNA ; not only because DNA was uppermost in my mind, but for several years
  DNA was also the only component of the telomeres that was identified. This
  was not for want of trying. I thought that DNA would not be the entire story of
  chromosome ends and, by extension from work emerging about chromatin
  in general in the 1970s, that it was likely that the telomeric DNA repeats
  tract would be packaged with proteins. The 1970s had seen great interest
  in chromatin, and the discovery of nucleosomes as the basic packaging
  unit of eukaryotic DNA. Telomeric sequences in <em>Tetrahymena </em>looked very
  intriguing to me in that regard, and as soon as I had identified the telomeric
  DNA I wanted to get my hands on whatever packaged it. Therefore, while still a
  postdoctoral fellow in Joe Gall's laboratory, I performed micrococcal nuclease
  treatment on isolated <i>Tetrahymena</i> nuclei. I found that the CCCCAA<sub>n</sub> tracts
  of the telomeres were protected in chromatin as a heterogeneous class
  of DNA fragments very different from that expected for nucleosomal
  packaging.</p>
<p>Soon after moving from Joe Gall's lab at Yale, while still temporarily at
  the University of California San Francisco (UCSF) in an independent research
  position (before I moved to the University of California Berkeley as
  an Assistant Professor), on March 1, 1978, I wrote to Joe Gall: &quot;I am getting
  quite excited about getting a CCCCAA<sub>n</sub>-binding protein complex from the
  <em>Tetrahymena </em>macronuclei, so I've been busily making rDNA, the CCCCAA<sub>n</sub>
  probe, and macronuclear micrococcal nuclease digests. Results so far are that
  I've found a simple salt fractionation that enriches for CCCCAA<sub>n</sub> sequence
  plus putative protein(s). The plan at the moment is to purify this some more
  so I can get some structural characteristics of any such complex, i.e. S value,
  and some identification of protein(s) in terms of 1-D and 2-D gel eletrophoretic
  properties ... The other aspect of course is to fish for something that will
  stick to a CCCCAA<sub>n</sub> column.&quot;</p>
  
<table width="350" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_5.jpg" width="350" height="470" alt="Scanned page from grant application, 1977."></td>
</tr>
<tr>
<td class="caption"><i>Figure 5</i>. Scanned page from grant application, 1977.</td>
</tr>
</table>
<p>&nbsp;</p>
<p>By 1980 I had done experiments to show that telomeric tracts of DNA
  in <i>Tetrahymena</i> were encapsulated in a protective sheath of protein that
  did not include nucleosomes. The vast majority of chromosomal DNA is
  packaged as nucleosomes: DNA-protein complexes. Each nucleosome
  is a flattened ball made up of histone proteins, around which the DNA
  is wrapped twice. The very basic (positively charged) histone proteins
  neutralize the negative charges of the phosphate chemical groups arrayed
  along the phosphodiester backbone of DNA and allow chromosomal
  DNA to become very closely packed and compactly folded in the nucleus.
  Nucleosomes in artificially stretched-out chromosomes are like beads on a
  string, although mostly in the nucleus they are closely packed into shorter
  thicker fibers. If one clips up chromatin using an enzyme, micrococcal
  nuclease, that cuts across the two strands of the linker DNA between
  neighboring nucleosomes, after getting rid of the histones, one can see that
  there are nucleosome-sized fragments of DNA left &ndash; a fragment of about
  142 base-pairs is protected by the histone core of the nucleosome, once
  the DNA linkers have been trimmed away. This kind of nuclease clipping
  behavior is a hallmark of a nucleosome. In contrast to nucleosomal regions of
  chromosomes, special regions of DNA, for example promoters that must
  bind transcription initiation factors that control transcription, have proteins
  other than the histones on them. The telomeric repeat tract turned out to
  be such a non-nucleosomal region. We found that if we clipped up chromatin
  using an enzyme that cuts the linker between neighboring nucleosomes,
  it cut up the bulk of the DNA into nucleosome-sized pieces but left the
  telomeric DNA tract as a single protected chunk. The resulting complex of
  the telomeric DNA tract plus its bound cargo of protective proteins behaved
  very differently, by various tests, from standard nucleosomal chromatin, and
  therefore we concluded that it had no histones or nucleosomes.</p>
<p>By 1977, it was known from work of Rekosh <i>et al</i>. that adenovirus DNA has a
  covalently bonded terminal protein, presumably for viral genome replication.
  Thus, in 1979, Marsha Budarf, a postdoctoral fellow in my laboratory at
  UC Berkeley, began using used radioactive iodine procedures (the Bolton-Hunter reagent) to see if we could find any comparable protein at the ends
  of rDNA. Although Marsha found a covalently attached protein (that in
  hindsight may have been topoisomerase I) enriched toward the end of the
  rRNA transcribed region, it was not enriched in the terminal parts of the
  rDNA molecules. She was unable to detect any other covalently attached
  protein elsewhere on the rDNA. Any evidence for a protein on the bulk of
  the rDNA molecule ends, such as their behavior in gel electrophoresis and
  the appearance of the rDNA molecules under the electron microscope, was
  conspicuously lacking. This made me feel all the more confident that there
  was no covalently attached protein at the very ends of this minichoromosome.
  But what other proteins were at telomeres?</p>
<p>My lab was the first to try to identify these protective proteins. We used
  biochemical fractionations of <i>Tetrahymena</i> nuclear extracts. My 1979 notebooks
  record that, together with my technician San-San Chiou in the
  Department of Molecular Biology at UC Berkeley, over and over I made
  attempts to purify the telomeric proteins from nucleoli. Nucleoli are the tiny
  bodies within the <i>Tetrahymena</i> nucleus that harbor the actively transcribed
  rDNA minichromosomes. Fractionations after fractionations, mostly using
  sucrose gradients, were patiently performed by San-San. Then we scaled
  up the preparations &ndash; I purchased a huge industrial-sized Waring blendor
  that loomed like a leviathan on the laboratory bench. <i>Tetrahymena</i> cells were
  blended in order to disrupt them just enough to shake their nucleoli free
  from the rest of the nuclear contents. At one time my note-book laconically
  reported: &quot;Respun only one-third of total ... Waring blender broke.&quot;</p>
<p>All these early efforts were to no avail. In retrospect, the experimental
  approach had been reasonable &minus; to purify nucleoli, as being the most enriched
  form of telomeric chromatin known, then to digest them with micrococcal
  nuclease into fragments, the end ones containing the telomeric DNA
  terminal tracts and their bound proteins. Then, I would further fractionate
  these away from the rest of the chromatin by selective precipitation in
  potassium chloride solutions, or fractionate them by size on sucrose
  gradients. The goal was to see what protein(s) would co-purify, through these
  multiple fractionation steps, with the telomeric repeat tract DNA, which I
  followed through the multiple steps by its hybridization signal. But we were
  only able to obtain limited amounts of chromatin and binding factors, and
  we tried without success to get enough to identify any factors that might be
  specific to the rDNA ends. Looking back, I see that we were fighting against
  the numbers game &ndash; our detection methods were too frail, our preparation
  scale-ups too modest. Therefore, it was yeast genetics and approaches done
  by others that turned out to provide the next great leaps forward in understanding
  telomeric proteins. That I failed in this by my early attempts using
  <i>Tetrahymena</i> made me all the more determined, if anything, to use other
  approaches to try to understand the nature and biological significance of
  those strange-seeming repeated sequences at the ends of chromosomes.</p>
<p>I also recall that our failure to find telomeric proteins taught a lesson that
  became useful when it came to our work on <i>Tetrahymena</i> telomerase. As Carol
  Greider's Nobel lecture describes, at one point the value of scaling up the
  telomerase activity preparations became evident to her. Thus, when Carol
  proposed the purchase of a very large glass column for preparative gel filtration
  chromatography, I was very willing to make this expensive-seeming purchase,
  ruefully recalling the past history of my too-pusillanimous scale-ups of
  <i>Tetrahymena</i> chromatin preparations.</p>
  
  <img src="blackburn_autobio_6.jpg" alt="Mission Bay Laboratory Group BBQ, August 2007">
<p class="photo-gallery-caption"><i>Figure 6</i>. Mission Bay Laboratory Group BBQ, August 2007.</p>  


<p>&nbsp;</p>
<p><strong>To the University of California San Francisco</strong><br>
  I became a Full Professor at UC Berkeley in 1986 (after 8 years on the faculty
  of UC Berkeley), and in the same year a mother (our son Benjamin David
  was born in December 1986). By around 1989, I decided that as the long
  drive to Berkeley each day from our home in San Francisco made it difficult
  to pursue both science and our family life optimally, it was time to begin
  investigating alternatives. I settled upon a professorship at UCSF, and
  the move of my laboratory to UCSF's Department of Microbiology and
  Immunology was accomplished in mid-1990. I have remained on the faculty
  of UCSF ever since. There, I have had the great good fortune to be able to
  keep delving into the nature and mechanisms of telomeres and telomerase.
  Together with colleagues in and out of UCSF and with my many talented
  students and postdoctoral fellows and technicians in my laboratory (Figure
  6), I have been able to address the wondrous biological systems comprised
  of telomeres and telomerase. A fanciful depiction evoking both telomere
  dynamics and telomere researchers is shown in Figure 7. This painting,
  done by the artist Julie Newdoll in 2008, elicits the idea of a telomere as an
  ancient Sumarian temple-like hive, tended by a swarm of ancient Sumarian
  Bee-goddesses against a background of clay tablets inscribed with DNA
  sequencing gel-like bands.</p>
<p>&nbsp;</p>
 <table width="350" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_7.jpg" width="350" height="444" alt="Julie Newdoll, Bee Goddesses"></td>
</tr>
<tr>
<td class="caption"><i>Figure 7</i>. Julie Newdoll, Bee Goddesses.</td>
</tr>
</table>
<p>&nbsp;</p>
<p><b>Out of the Laboratory</b><br>
  In the 1990s my research's implications for humans began to intrigue me,
  but with scientific research, faculty and Department Chair duties, family and
  many associated commitments, I had little time to indulge in delving into
  the philosophical and policy questions that can arise as science opens new
  possibilities. I served as President of the American Society for Cell Biology
  in 1998 and become more cognizant of the world of national science policy.
  Thus it was that in late 2001, the request to consider becoming a member
  of a newly created U.S. Federal Commission, the President's Council on
  Bioethics, had a certain appeal. I felt that my knowledge of the relevant fields
  of science, and long experience in the world of research, would be useful
  contributions to the Council, a body that, as a Federal Commission, would
  be advisory on some matters of national science policy. A further appeal was
  the coincidence with my growing thinking about these issues. I reasoned that
  if I joined this Council, it would be an opportunity to contemplate some of
  these dimensions of research's ramifications, and the possible reverberations
  of my own area of research.</p>
<p>Time for quiet contemplation of these and related questions in the
  abstract was not forthcoming. I understood from the beginning that the
  Bioethics Council would be occupied with publicly debated topics including
  human somatic cell nuclear transfer and embryonic stem cell research,
  as well as other topics less clearly defined at the outset of the council's
  deliberations. I thought I should agree to serve on this Council because, as
  a seasoned scientist (particularly in cell and molecular biology), I might be
  able to offer perspectives that would be helpful in advising national scientific
  policy. I knew the topics upon which this Council, appointed by the George
  W. Bush administration, would advise would be politically charged ones. For
  this reason especially, I felt that a strong base of scientific fact and evidence
  would be particularly important, and useful advice in this vein was something
  that I could in fact offer to this advisory body.</p>
<p>I publicly made clear my views on some of the council's recommendations,
  views that did not generally accord with those of the White House or with
  those of the Council's Chair. After two years, I was informed by the Personnel
  Office of the George W. Bush White House that I would no longer be on
  this Council. This dismissal from the Council received quite a lot of public
  attention at the time. In the course of it, I was overwhelmed by the great
  many letters and communications I received. Almost without exception
  positive and supportive, they came from all over the United States and even
  from as far afield as a musician in London. His somewhat (to me) unexpected
  concern for science policy brought home to me how widespread is the
  wish among the public that science policy be informed by good scientific
  evidence. This entire episode was a broadening education. It reinforced my
  love of the searches for truth to which so many in research and academia
  aspire.</p>
<p><b>People Who Have Had Important Influences on My Life as a Scientist</b><br>
  I am indebted to so many individuals that I can only describe a few of them
  here. Growing up, three of my schoolteachers in particular encouraged
  my interests in biology and chemistry and mathematics, not least by letting
  me know that they believed in my abilities to succeed in these areas &ndash; Nan
  Hughes, Jenny Phipps and Len Stuttard.</p>
<p>As I embarked on research in biological science, my teachers, advisors
  and mentors &minus; notably Frank Hird in Australia, Fred Sanger in Cambridge,
  England, and Joe Gall in the U.S.A. &minus; not only imparted their scientific knowledge,
  visions and wisdom, but also their examples of how to be a scientist. In
  particular, a photograph of Joe Gall from 1999, although taken several years
  after I had been in his lab, captures in a succinct visual way some of Joe's
  characteristics that influenced me when I was a member of his lab group
  (Figure 8). I took the photograph during a conference he was attending in
  Prague in the summer of 1999. During the conference a partial eclipse of the
  sun took place, and all the conference participants rushed out of the lecture
  hall to witness its progress. Joe is seen in the photograph demonstrating that
  it could be seen very simply and safely: All one had to do was hold a flat sheet
  of paper under a leafy bush so that the light, diffracted through the leaves
  onto the paper, caused to appear on the sheet of paper images of the &quot;bite&quot;
  being taken out of the disc of the sun by the moon passing in front of it. I
  recall that most of the conference participants had never seen this applied
  optics demonstration before. This photo evokes at once Joe Gall's desire and
  ability to teach &minus; by his use of a very striking demonstration to teach something
  new to the conference participants &minus; and, not least, one glimpse of his
  wide knowledge encompassing optics and natural science in general. </p>
  
<table width="255" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_8.jpg" width="255" height="385" alt="Joseph G. Gall. Prague, 1999"></td>
</tr>
<tr>
<td class="caption"><i>Figure 8</i>. Joseph G. Gall. Prague, 1999.</td>
</tr>
</table>
 <p>&nbsp;</p>
<p>Like so many who are fascinated by chromosome behavior, I owe much to
  Barbara McClintock for her scientific findings. But in addition, <a href="/nobel_prizes/medicine/laureates/1983/">Barbara
  McClintock</a> also gave me a memorable lesson: in a conversation I had with
  her in 1977, during which I had told her about my unexpected findings
  with the rDNA end sequences, she urged me to trust my intuition about my
  scientific research results. This advice was surprising to me then, because
  intuitive thinking was not something that at the time I allowed myself to
  admit might be a valid aspect of being a biology researcher. I think her advice
  recognizes an important and sometimes overlooked aspect of the intellectual
  processes that underlie scientific research, and for me it had a liberating
  aspect to it. For this, also, I am very grateful to Barbara McClintock.</p>
<table width="350" cellpadding="2">
<tr>
<td><img src="blackburn_autobio_9.jpg" width="350" height="504" alt="Elizabeth and her son, Ben, at the piano. Circa 1990"></td>
</tr>
<tr>
<td class="caption"><i>Figure 9</i>. Elizabeth and her son, Ben, at the piano. Circa 1990.</td>
</tr>
</table>
 <p>&nbsp;</p>
 <p>My husband, John Sedat, himself an accomplished scientist, has always
   urged me to dig deeper into myself and find the reserves of strength I might
   
   not have tapped &ndash; his encouragement in this way has helped me through
   years of doing science. Our son Ben (Figure 9) inspired me to try to find
   ways of combining family and science, something that I have tried to convey
   to young scientists making their careers. Finally, my parents were both family
   physicians. From them I imbibed a sense of the importance of serving people
   kindly and as well as one can. I continue to believe that bioethics, done
   well and underpinned by the best available scientific evidence, can be an
   important part of our consideration, as a society, of the impact on people of
   scientific research in the biological sciences and medicine.</p>
