  <h1>Roger D. Kornberg - Biographical</h1>
<p><img src="kornberg.jpg" class="bio-img">
My adult scientific career began with graduate study in chemical physics with
Harden McConnell at Stanford. I had the idea of elucidating the mechanism
of ion transport across biological membranes by nuclear resonance. I thought
ion transport must involve rotation of the transport protein in the membrane.
Struggling to prove this wrong idea, it occurred to me to study the rotation in
the membrane of a lipid molecule, about 1,000 molecular weight, rather than
a protein fifty times larger. This led to my discoveries, by nuclear and paramagnetic
resonance methods, of phospholipid flip-flop, an exceedingly slow
process, and lateral diffusion, exceedingly fast (Kornberg and McConnell,
1971a ; Kornberg and McConnell, 1971b).</p>

<p>For postdoctoral work, I wanted to learn about the other important method
of physico-chemical analysis of macromolecules, X-ray diffraction. The obvious
choice was the Laboratory of Molecular Biology (LMB) in Cambridge, where
protein crystallography was developed and still most intensively practiced at
the time. I went in the spring of 1972 to work with <a href="/nobel_prizes/chemistry/laureates/1982/index.html">Aaron Klug</a>, who was not
only a leading crystallographer, but also responsible for the application of
Fourier methods to electron microscopy and image processing. While looking
for a problem to study by X-ray diffraction, I got to know Mark Bretscher, the
only person at the LMB interested in membrane structure, and he suggested
reading a paper just published by <a href="/nobel_prizes/medicine/laureates/1962/index.html">Francis Crick</a> titled &quot;A General Model for
Higher Organism Chromosomes&quot; (Crick, 1971). Figure 3 of that paper was a
diagram showing a loop of DNA crossed by a dashed line, said to symbolize a
histone molecule. When I raised the subject with Aaron Klug, he immediately
produced a sheaf of papers on the X-ray analysis of chromosomal material,
or &quot;chromatin,&quot; known for nearly a century to contain roughly equal weights
of histones and DNA. Aaron had discussed the interpretation of the X-ray
pattern of chromatin extensively with Francis, and he encouraged me to pursue
the problem. He warned me, however, that it was a &quot;messy&quot; problem.</p>
<p>Notorious might have been a better word. Many had succumbed to the allure
  of the problem, with its potential for insight into genetic chemistry, only
  to be frustrated by the intractability of the histones. These proteins were,
  on the one hand, surprisingly simple, and on the other hand, hopelessly
  complicated. There are only five types of histone, designated H1, H2A, H2B,
  H3, and H4. Upon isolation, however, the individual histones proved to be
  extraordinarily sticky, binding avidly to DNA and interacting with one another
  in every possible combination. Whereas the X-ray diffraction pattern of
  chromatin was indicative of repeating order, the biochemical behavior of the
  histones did not appear to explain it. There was, moreover, sufficient variation
  in the relative amounts of the histone types in various tissues and organisms
  &quot;to make the idea of a unique repeating order untenable&quot; (Huberman,
  1973). The histones came to be regarded as a kind of amorphous glue, coating
  the chromosomal DNA, with no obvious significance.</p>
<p>I began by repeating the work of others, isolating the individual histones,
mixing them in various combinations with DNA, and recording X-ray diffraction
patterns. I also scoured the literature and came across two papers
that influenced my thinking. A paper by Hewish and Burgoyne reported the
cleavage of about 10% of chromosomal DNA by an endogenous nuclease
in isolated rat liver nuclei to multiples of a unit size (Hewish and Burgoyne,
1973). When I mentioned this to Francis Crick, he shot off a letter to Hewish
inquiring about the size. The reply came from Burgoyne, giving the sedimentation
coefficient of the unit length of DNA. Assuming the measurement was
made under alkaline conditions, as was customary for sedimentation analysis
of RNA, much studied at the LMB at the time, the value from Burgoyne
corresponded to about 500 bases. This unit size did not relate to any other
information about chromatin, and the appearance of multiples of the unit
size simply confirmed what we already knew from X-ray diffraction, that chromatin
contained some amount of repeating substructure.</p>

<p>The second paper, by van der Westhuyzen and von Holt, reported the
extraction of histones from chromatin by mild methods, rather than with
strong acid or other harsh treatment, as was customary at the time (van der
Westhuyzen and von Holt, 1971). Mild methods failed to resolve the histones
entirely from one another, so the paper was ignored. What attracted my attention
was the clean separation of the mildly extracted histones into two
groups, H2A/H2B, and H3/H4. This separation contrasted with the promiscuous
interactions of the histones previously observed. I realized this promiscuity
was likely attributable to the denaturation of histones during isolation
in the past. From the data in the paper, I could also deduce that the H3/H4
group behaved as if twice the size of the H2A/H2B group, although all four
individual histone proteins were about the same size. I concluded that H3
and H4 must form a dimer, and I thought I might crystallize and solve the
structure of this unique histone oligomer.</p>

<p>What followed was truly astounding. I measured the molecular weight of
the purified H3/H4 preparation by equilibrium ultracentrifugation, while
Jean Thomas offered to analyze the material by chemical cross-linking. Both
methods showed unequivocally that H3 and H4 were in the form of a double
dimer, an (H3)<sub>2</sub>(H4)<sub>2</sub> tetramer (Kornberg and Thomas, 1974). I pondered
this result for days, and came to the following conclusions (Kornberg, 1974).
First, the exact equivalence of H3 and H4 in the tetramer implied that the differences
in relative amounts of the histones from various sources measured
in the past must be due to experimental error. This and the stoichiometry of
the tetramer implied a unit of structure in chromatin based on two each of
the four histones, or an (H2A)<sub>2</sub>(H2B)<sub>2</sub>(H3)<sub>2</sub>(H4)<sub>2</sub> octamer. Second, since
chromatin from all sources contains roughly one of each histone for every
100 bp of DNA, a histone octamer would be associated with 200 bp of DNA.
Finally, the (H3)<sub>2</sub>(H4)<sub>2</sub> tetramer was reminiscent of hemoglobin, an <span class="style1">a</span><sub>2</sub><span class="style1">b</span><sub>2</sub>
tetramer. The X-ray structures of hemoglobin and other oligomeric proteins
available at the time were compact, with no holes through which a molecule
the size of DNA might pass. Rather, the DNA in chromatin must be wrapped
on the outside of the histone octamer.</p>

<p>As I turned these ideas over in mind, it struck me how I might explain the
results of Hewish and Burgoyne. What if their sedimentation coefficient of
unit length DNA fragments was measured under neutral rather than alkaline
conditions? Then the DNA would have been double stranded and about 250
bp in length. Allowing for the approximate nature of the result, the correspondence
with my prediction of 200 bp was electrifying. Then I recalled a
reference near the end of the Hewish and Burgoyne paper to a report of a
similar pattern of DNA fragments by Williamson. I rushed to the library and
found that Williamson had obtained a ladder of DNA fragments from the
cytoplasm of necrotic cells and measured the unit size by sedimentation under
neutral conditions: the result was 205 bp! I was euphoric. In the months
and years to follow, it was often pointed out how thin was the support for my
ideas and how extended the line of reasoning, but I never really doubted the
conclusions. The prediction of the DNA unit size and its verification convinced
me completely.</p>

<p>Support for a particulate substructure of chromatin came from electron
microscopy and from nuclease digestion and sedimentation analysis. Some
work on these lines was done even before my own, and though not definitive,
was nicely coincident with my ideas. In the years to follow, with colleagues in
Cambridge, I proved the existence of the histone octamer and the equivalence
of the 200 bp unit with the particle seen in the electron microscope
(Kornberg, 1977). This chapter of the chromatin story concluded with the
X-ray crystal structure determination of the particle, now known as the nucleosome,
showing a histone octamer surrounded by DNA, in near atomic
detail (Luger <i>et al</i>., 1997).</p>

<p>At this natural break in the work, I returned to the US, first as faculty member
of the Department of Biological Chemistry of Harvard Medical School
in 1976, and then the Department of Structural Biology of Stanford Medical
School in 1978. I had decided to pursue the function rather than the structure
of the nucleosome, and was joined in this by Yahli Lorch, who became
my lifelong partner in chromatin research, and also my partner in life. We
investigated the consequences of the nucleosome for transcription. It was
believed that histones are generally inhibitory to transcription. We found, to
the contrary, that RNA polymerases are capable of reading right through a
nucleosome. Coiling of promoter DNA in a nucleosome, however, abolished
initiation by RNA polymerase II (pol II) (Lorch <i>et al.</i>, 1987). This finding, together
with genetic studies of Michael Grunstein and colleagues, identified a
regulatory role of the nucleosome in transcription. It has since emerged that
nucleosomes play regulatory roles in a wide range of chromosomal transactions.
A whole new field has emerged, one of the most active in bioscience
today. It involves a bewildering variety of posttranslational modifications of
the histones, and a protein machinery of great complexity for applying, recognizing,
and removing these modifications.</p>

<p>Although Yahli's first priority was the rearing of our children, Guy, Maya,
and Gil, she performed a series of important studies of chromatin remodeling,
believed to oppose inhibition by nucleosomes. She began with Brad
Cairns, who discovered the RSC chromatin remodeling complex. Subsequent
work showed that RSC disrupts nucleosome structure and slides histone octamers
along the DNA (Lorch <i>et al.</i>, 1998; Lorch <i>et al.</i>, 2001). Most recently,
we found that RSC can transfer a histone octamer to a histone chaperone
protein, exposing the nucleosomal DNA for transactions such as transcription
(Lorch <i>et al.</i>, 2006).</p>

<p>For studies of transcriptional regulation, it was necessary to identify the
transcription machinery. Biochemical work, in the laboratories of Robert
Roeder, Ronald and Joan Conaway, and others, revealed the complexity of
the RNA polymerase II transcription machinery of mammalian cells, and it
was apparent there was much to be gained by studies in yeast. A combined
genetic and biochemical approach, possible only in yeast, would be advantageous
for the solution of such a difficult problem. There was, however, a
major obstacle. The development of a yeast pol II transcription system had
been attempted unsuccessfully in many laboratories for more than a decade.
Efforts to develop biochemical systems from yeast for RNA splicing, membrane
transport, and other processes had also met with failure, leading to the
widely held view that yeast, although the organism of choice for eukaryotic
genetic analysis, was unsuited to biochemical investigation. First Yahli, and
then graduate student Neal Lue, took up the challenge. Neal was eventually
successful, by changing a number of procedures and conditions used
in the past, including the replacement of chloride by acetate ion (Lue and
Kornberg, 1987). Remarkably, chloride, though a component of other transcription
systems, is highly inhibitory to yeast pol II transcription.</p>

<p>Neal, Ray Kelleher, and Peter Flanagan undertook fractionation of the
yeast system, followed by Mike Sayre, John Feaver, Herbert Tschochner,
Opher Gileadi, and Lynn Henry. The yeast system was problematic, since the
starting transcription signal was a thousand-fold less than that in mammalian
systems, and we soon reached an impasse. After enriching the transcription
proteins about a hundred-fold through two chromatographic steps, we could
proceed no further. Every subsequent step eliminated transcription completely.
Mike achieved a breakthrough by starting over. He realized the problem
was the loss of essential components and an accumulation of inhibitors,
and he devised a new fractionation procedure that was ultimately successful
(Sayre <i>et al.</i>, 1992).</p>

<p>An important outcome of this work was the discovery that yeast and
mammalian pol II systems are the same. Differences in promoter structure
believed to reflect fundamental differences in the transcription proteins
proved to be insignificant. The real payoff, however, was in the study of transcriptional
regulation. In the late 1980s, it was thought the communication
between gene activator proteins and the transcription machinery was direct.
In 1990 and 1991, Ray and Peter produced evidence for an additional factor
required for this communication in a still crude yeast system (Flanagan <i>et al.</i>,
1991; Kelleher <i>et al.</i>, 1990). This additional factor, which we referred to as
Mediator, proved elusive. The requirement for the factor was variable over
the course of fractionation. As so often happens, the solution of the problem
came from an unexpected direction. The most difficult protein of the pol II
transcription system to purify, known as factor IIH, was contaminated with
Mediator, and only after entirely pure IIH was obtained by Jesper Svejstrup
(Svejstrup <i>et al.</i>, 1995) were Stefan Björklund and Young-Joon Kim then able
to complete the isolation of Mediator (Kim <i>et al.</i>, 1994). To our astonishment,
pure Mediator proved to be an assembly of nearly two dozen proteins,
more than half of which were known from genetic studies in yeast to be involved
in transcriptional regulation.</p>

<p>In parallel with these biochemical studies of pol II transcription, we pursued
the structure of the transcription machinery. Pol II, the central component
of the machinery, is an assembly of a dozen proteins, several times the
size of any asymmetric structure determined by X-ray diffraction in the early
1980s when we began. The story of our solution of the pol II problem is told
in my <a href="/nobel_prizes/chemistry/laureates/2006/kornberg-lecture.html">Nobel Lecture</a>, except for its origins in my graduate studies of lipid diffusion.
While I was in Cambridge, Aaron Klug and collaborators were developing
electron microscope crystallography for the solution of protein structures
too large for X-ray crystallography. As the name implies, the electron
microscope approach also requires protein crystals, only they must be thin,
preferably a single protein molecule thick, or &quot;two-dimensional,&quot; for transmission
of the electron beam. The work in Cambridge at the time was limited
to naturally occurring ordered arrays of proteins, and it was apparent that
a general method of forming two-dimensional crystals was needed to bring
any protein of interest within reach of the electron microscope procedure.
I thought I might exploit lateral diffusion in lipid layers for the purpose.
A protein bound to a lipid layer would be constrained in two dimensions
but free to diffuse and, I hoped, to crystallize. I tried this idea, beginning
with nucleosomes and positively charged lipid layers. Finally, after repeated
failures over a period of about five years, a sabbatical visitor, Ed Uzgiris, was
successful, with the combination of a monoclonal antibody directed against
a lipid hapten (Uzgiris and Kornberg, 1983). This work led eventually to twodimensional
crystals of pol II, and then to large single crystals and the X-ray
structure of pol II.</p>

<p>Our biochemical and crystallographic work converged in the X-ray structure
determination of pol II. This convergence was the culmination of a
long effort, but also a beginning. Pol II associates with two dozen additional
transcription proteins and with Mediator in a giant assembly formed at every
promoter prior to the initiation of transcription. The ultimate goal of our
work is to solve this giant complex and thus to understand the mechanism
and regulation of transcription. We believe we will achieve this goal within
the next decade, through crystallography and, possibly, another development in electron microscopy. Inspired by graduate student Grant Jensen, we
have pursued the synthesis of large heavy atom clusters, for the purpose of
structure determination without crystals at near atomic resolution (Jensen
and Kornberg, 1998). This work has opened a window on a whole new realm
of inorganic chemistry and materials science, through which we may pass
into the future.</p>
<table cellpadding="2" summary="References">
  <tr>
    <td><b>References</b></td>
  </tr>
  <tr>
    <td>Crick, F. (1971). General model for the chromosomes of higher organisms. Nature <i>234</i>,
      25-27.</td>
  </tr>
  <tr>
    <td>Flanagan, P. M., Kelleher III, R. J., Sayre, M. H., Tschochner, H., and Kornberg, R. D.
      (1991). A mediator for activation of RNA polymerase II transcription <i>in vitro</i>. Nature
      <i>350</i>, 436-438.</td>
  </tr>
  <tr>
    <td>Hewish, D. R., and Burgoyne, L. A. (1973). Chromatin sub-structure. The digestion of
chromatin DNA at regularly spaced sites by a nuclear deoxyribonuclease. Biochem.
Biophys. Res. Commun. <i>52</i>, 504-510.</td>
  </tr>
  <tr>
    <td>Huberman, J. A. (1973). Structure of chromosome fibers and chromosomes. Annu. Rev.
Biochem. <i>42</i>, 355-378.</td>
  </tr>
  <tr>
    <td>Jensen, G. J., and Kornberg, R. D. (1998). Single-particle selection and alignment with
heavy atom cluster-antibody conjugates. Proc. Natl. Acad. Sci. USA <i>95</i>, 9262-9267.</td>
  </tr>
  <tr>
    <td>Kelleher, I., R.J., Flanagan, P. M., and Kornberg, R. D. (1990). A novel mediator between
activator proteins and the RNA polymerase II transcription apparatus. Cell <i>61</i>, 1209-
1215.</td>
  </tr>
  <tr>
    <td>Kim, Y. J., Björklund, S., Li, Y., Sayre, M. H., and Kornberg, R. D. (1994). A multiprotein
mediator of transcriptional activation and its interaction with the C-terminal repeat domain
of RNA polymerase II. Cell <i>77</i>, 599-608.</td>
  </tr>
  <tr>
    <td>Kornberg, R. D. (1974). Chromatin structure: a repeating unit of histones and DNA.
Science <i>184</i>, 868-871.</td>
  </tr>
  <tr>
    <td>Kornberg, R. D. (1977). Structure of chromatin. Ann. Rev. Biochem. <i>46</i>, 931-954.</td>
  </tr>
  <tr>
    <td>Kornberg, R. D., and McConnell, H. M. (1971a). Inside-outside transitions of phospholipids
in vesicle membranes. Biochemistry <i>10</i>, 1111-1120.</td>
  </tr>
  <tr>
    <td>Kornberg, R. D., and McConnell, H. M. (1971b). Lateral diffusion of phospholipids in a
vesicle membrane. Proc. Natl. Acad. Sci. USA <i>68</i>, 2564-2568.</td>
  </tr>
  <tr>
    <td>Kornberg, R. D., and Thomas, J. O. (1974). Chromatin structure; oligomers of the histones.
Science <i>184</i>, 865-868.</td>
  </tr>
  <tr>
    <td>Lorch, Y., Cairns, B. R., Zhang, M., and Kornberg, R. D. (1998). Activated RSC-nucleosome
complex and persistently altered form of the nucleosome. Cell <i>94</i>, 29-34.</td>
  </tr>
  <tr>
    <td>Lorch, Y., LaPointe, J. W., and Kornberg, R. D. (1987). Nucleosomes inhibit the initiation
of transcription but allow chain elongation with the displacement of histones. Cell <i>49</i>,
203-210.</td>
  </tr>
  <tr>
    <td>Lorch, Y., Maier-Davis, B., and Kornberg, R. D. (2006). Chromatin remodeling by nucleosome
disassembly <i>in vitro</i>. Proc. Natl. Acad. Sci. USA <i>103</i>, 3090-3093.</td>
  </tr>
  <tr>
    <td>Lorch, Y., Zhang, M., and Kornberg, R. D. (2001). RSC unravels the nucleosome. Mol. Cell
      <i>7</i>, 89-95.</td>
  </tr>
  <tr>
    <td>Lue, N. F., and Kornberg, R. D. (1987). Accurate initiation at RNA polymerase II promoters
in extracts from <i>Saccharomyces cerevisiae</i>. Proc. Natl. Acad. Sci. USA 84, 8839-8843.</td>
  </tr>
  <tr>
    <td>Luger, K., Mader, A. W., Richmond, R. K., Sargent, D. F., and Richmond, T. J. (1997). Crystal
structure of the nucleosome core particle at 2.8 A resolution. Nature <i>389</i>, 251-260.</td>
  </tr>
  <tr>
    <td>Sayre, M. H., Tschochner, H., and Kornberg, R. D. (1992). Reconstitution of transcription
with five purified initiation factors and RNA polymerase II from <i>Saccharomyces cerevisiae</i>.
J. Biol. Chem. <i>267</i>, 23376-23382.</td>
  </tr>
  <tr>
    <td>Svejstrup, J. Q., Wang, Z., Feaver, W. J., Wu, X., Bushnell, D. A., Donahue, T. F., Friedberg,
E. C., and Kornberg, R. D. (1995). Different forms of TFIIH for transcription and DNA
repair: Holo-TFIIH and a nucleotide excision repairosome. Cell <i>80</i>, 21-28.</td>
  </tr>
  <tr>
    <td>Uzgiris, E. E., and Kornberg, R. D. (1983). Two-dimensional crystallization technique for
imaging macromolecules, with an application to antigen-antibody-complement complexes.
Nature <i>301</i>, 125-129.</td>
  </tr>
  <tr>
    <td>van der Westhuyzen, D. R., and von Holt, C. (1971). A new procedure for the isolation and
fractionation of histones. FEBS Letters <i>14</i>, 333-337.</td>
  </tr>
</table>
