<h1>Jack W. Szostak - Biographical</h1>

<p><img src="szostak.jpg" class="bio-img">I greatly enjoy reading the biographies of scientists, and when doing so I
always hope to learn the secrets of their success. Alas, those secrets generally
remain elusive. Now that I find myself in the surprising situation of having
to write my own biography, and thus to reflect on my career, I find the same
mystery. I do not know why I have always been fascinated by science, or why I
have been driven by the intense desire to make some original contribution.
And although I have had some degree of success as a scientist, it is hard to
say precisely why. Nevertheless, I have attempted to identify some of the
incidents and decisions that helped or hindered me at various times, in
the hope that these anecdotes might be helpful to those embarking on a
scientific career.</p>
<p>I have generally sought to work on questions that I thought were both
interesting and approachable, yet not too widely appreciated. To struggle to
make discoveries that would be made by others a short time later seems futile
to me. This, coupled with a distaste for direct competition, attracts me to
areas of science that are less densely populated. On multiple occasions, I
have been led into these new areas by talking to people working in fields
quite different from mine. The confluence of ideas from distinct fields
seems to create a kind of intellectual turbulence that is both exciting and
productive.</p>
<p>My knowledge of the details of my family history is rather sketchy. My
paternal great-grandfather was born near Cracow, and emigrated
to New York City in the late 19th century, but ultimately settled in
a small farming town in Saskatchewan, Canada where my father
was born. Eager to escape the small town isolation, my father left as
soon as he could by joining the Royal Canadian Airforce (RCAF)
towards the end of World War II. He was trained as a pilot but fortunately
the war ended before he could serve in combat, and he was then posted
to Ottawa. My mother's family came from England but settled in Ottawa,
where my mother was raised and met my father after the war. Shortly
after they married my parents moved to England for my father's continued
training in aeronautical engineering at Imperial College, London. I was
born in London, England during the great fog of 1952, but survived the
coal-fueled air pollution with no ill effects and after less than a year in
England was carried to Canada by my parents. My father continued to work
as an aeronautical engineer for the RCAF for the next twenty years, and our
house was always decorated with models of the airplanes he worked on. After
he retired my father joined the civil service, and for a time studied issues
of Arctic transportation; I remember him telling me about the complex properties of Arctic sea ice. Some of my work has an engineering flavor, in
that we build structures and test their properties, and it's possible that it may
reflect some influence of my early home life.
But a more direct influence
stems from the fact that my father was often unhappy with his job, chafing at
both his superiors and his subordinates. This I am sure made me seek out the
academic life for its more egalitarian aspects. I have never felt like I worked
for a boss or had employees who worked for me, just colleagues who like me
were interested in learning more about the world around us.</p>

 <img src="szostak_autobio_1.jpg" alt="Family photos">
<p class="photo-gallery-caption"><i>Figure 1</i>. Family photos. 1A: Photo taken by my mother (Vi Szostak) on the HMS Homeric,
as we returned to Canada in 1960 after three years on the RCAF base in Zweibr√ºcken,
Germany. Left to right, me, my mother and sisters Kathy and Carolyn. 1B: Photo taken by
my mother, in Ottawa, 1963, showing me, my father (Bill Szostak), and sisters Carolyn and
Kathy, with the Canadian Parliament Buildings in the background.</p>



 <p>&nbsp;</p>
 
 <p>My childhood was punctuated by frequent moves, as my father was
transferred to different Air Force postings in Germany, Montreal, and Ottawa
(Figure 1). At the time many school systems encouraged students to advance
as rapidly as possible; as a consequence I was often the youngest in my class.
Although socially difficult, this was more than compensated for by making
my classes more interesting than they would otherwise have been. Some of
my earliest recollections involve grade school math. Learning about fractions
was for some reason surprising enough to have stuck with me for the rest
of my life; similarly, my discovery of quadratic equations in grade 5 was a
revelation. Later, at Riverdale High School in suburban Montreal I was
fortunate to have some exceptional teachers. Don Hall struggled to answer
my strange science questions, and Irene Brun (now Winston) inspired a
life-long love of biology. At the same time, my interest in science was encouraged
at home. My father built a basement chemistry lab for me, and the
experiments I conducted there often made use of remarkably dangerous
chemicals that my mother was able to bring home from the company where
she worked. My mother also helped me to get my first summer job, in a
chemical testing laboratory at the same company. This was a good window
into the importance of quantitative analysis, but the repetitive nature of
the work was not at all interesting. Some of the experiments carried out in my basement lab were much more dramatic. For example, with my
father's assistance, shortly after the tragic Apollo 1 fire, we prepared and
collected a jar of pure oxygen. We then carefully lowered a small quantity of
methanol into the oxygen reservoir. The transformation of the barely visible
pale blue flame in air into an intense jet of fire in oxygen was amazing, but
also horrifying in the context of the recent Apollo fire. Less carefully supervised
experiments frequently led to explosions, which made chemistry seem
much more dramatic than one would guess from the textbooks. My failure to
carefully separate the hydrogen evolved during electrolysis from ambient air
led to an impressive explosion which resulted in a glass tube being embedded
in a wooden ceiling rafter. I also participated in more biologically oriented
projects with my high school friend Joachim Sparkuhl. In the basement of his
house we constructed a small hydroponics garden, inspired, I believe, by the
idea that astronauts living on some future space station would need or want
to grow their own fresh food.</p>
 <p>In 1968 I began my undergraduate studies at McGill, at the age of 15. My
 first laboratory work at McGill involved helping a chemistry graduate student
 to purify cholesterol, the starting material for the synthesis of sterols. We
 started with large sacks of gallstones, which we would dissolve in hot solvent,
 and then recover the iridescent crystals of pure cholesterol after the solution
 cooled. While this was a useful experience, it did not inspire me to remain
 in chemistry, and the pull of biology increased as new opportunities opened
 up. To my surprise I was accepted into a summer research program
 for undergraduates at the Jackson Laboratories, a renowned mouse
 genetics institute on Mt. Desert Island off the coast of Maine. The
 environment was idyllic, and the program combined intense scientific
 education and hands-on experimental work with outdoor activities such as
 hiking up Cadillac Mountain and observing the beautiful organisms that
 populated the nearby tidal pools. The Jackson labs are a mouse genetics research
 facility, and this influenced my future scientific career in an unexpected way.
 My project, carried out under the guidance of Dr. Chen K. Chai, involved
 the analysis of thyroid hormones in various mutant strains. This required the
 careful dissection of the thyroid gland from many mice. Although I was, after
 much practice, able to remove the thyroid without (at least most of the time)
 severing any of the many nearby major blood vessels, I strongly disliked the
 process of killing and dissecting the animals, and by the end of the summer
 had vowed never again to work on animal models.</p>
 <p>Back at McGill the next fall, this time as a resident student (my parents
 having moved back to Ottawa), I started spending less time in the lectures
 and more time in the library, and also searching out new labs in which to gain
 additional experience. I was always surprised when seemingly intimidating
 Professors welcomed me into their labs and invited me to join in ongoing
 research projects. During this year and the next I did work in several labs
 in the Biology and Biochemistry departments, generally on plant biology
 systems. Field trips with Kurt Meier, a specialist in bryophyte biology, inculcated
 an enduring affection for the simple mosses and liverworts. I apparently did well enough in a physiology course run by Ron Poole to land a summer
 job prototyping and testing new lab experiments for the following year's
 lab course. Although most of my lecture courses were uninspiring, John
 Southin's course in Molecular Biology was an incredible exception. I'll never
 forget entering the first class and being handed a thick book of printouts,
 which I assumed were a set of papers we were supposed to read. In fact the
 whole book was simply a list of references, which we were expected to read
 and absorb in the library. These readings from the frontiers of molecular
 biology were very impressive. We read and discussed the beautiful Meselson-Stahl experiment, which was just over a decade old at the time, and learned
 how the genetic code had been unraveled only a few years previously. The fact
 that one could deduce, from measurements of the radioactivity in fractions
 from a centrifuge tube, the molecular details of DNA replication, transcription
 and translation was astonishing to me. One of the intellectual highlights
 of my time at McGill was the open-book, open-discussion final exam in this
 class, in which the questions were so challenging that the intense collaboration
 of groups of students was required to reach the answers.</p>
 <p>In my senior year, I began a project in Mel Goldstein's lab, together with
 my friend Joachim Sparkhul. The subject of our study was the beautiful
 colonial green flagellate <em>Eudorina elegans</em>, a smaller version of the more
 common <em>Volvox</em>. Over the school year and the following summer, we obtained
 evidence that these algae secreted a peptide hormone that induced
 spermatogenesis under favorable environmental conditions. This work led to
 our first scientific publication, which appeared the following year (1).</p>
 <p>In the fall of 1972 I started my graduate studies at Cornell University in
 Ithaca, New York. I decided to attend Cornell in part because the A.D. White
 Fellowship would fully support me, but also because I would be able to
 pursue my work on <em>Eudorina </em>in the Department of Plant Physiology. At the
 time, I was enamored with a grandiose plan to develop <em>Eudorina </em>as a simple
 model system for studies in developmental genetics. This plan did not work
 out, for several reasons, not least the fact that this sort of ambitious program
 cannot be developed in isolation by an inexperienced student. Lacking the
 necessary genetic expertise, and because I was either unable or unwilling
 to seek out the necessary help, my project became mired in frustrating
 technical difficulties.</p>
 <p>However, the periods spent waiting for my <em>Eudorina </em>cultures to grow
 allowed for plenty of time for conversations with my fellow graduate student
 John Stiles. John was approaching graduation and was thinking about what
 to do after the completion of his Ph.D., while I was gradually shifting from
 thinking about <em>Eudorina </em>to dreaming up some more productive project. We
 talked a lot about the emerging methods in molecular biology, which were
 clearly heading towards the ability to explore the structure and activity of
 individual genes at the molecular level; cloning and sequencing technologies
 were just beginning to emerge. John and I eventually came up with a
 specific proposal for a collaborative experiment. Our idea was to chemically
 synthesize a DNA oligonucleotide of sufficient length that it would hybridize to a single sequence within the yeast genome, and then to use it as an
 mRNA and gene specific probe. While conceptually simple, our idea was
 technically challenging. At the time, there was only one short segment of the
 yeast genome for which the DNA sequence was known, the region coding for
 the N-terminus of the iso-1 cytochrome c protein, which had been intensively
 studied by Fred Sherman for many years. The Sherman lab, in a tour de force
 of genetics and protein chemistry, had isolated double-frameshift mutants
 in which the N-terminal region of the protein was translated from out-of-frame
 codons. Protein sequencing of the wild type and frameshifted mutants
 allowed them to deduce 44 nucleotides of DNA sequence. John and I thought
 that if we could prepare a synthetic oligonucleotide that was complementary
 to the coding sequence, we could use it to detect the cytochrome-c mRNA
 and gene. At the time, essentially all experiments on mRNA were done on
 total cellular mRNA, rendering efforts to monitor the expression of
</p>
 <p>John and I were sufficiently confident of our ideas to begin contacting labs
 where we might pursue the work, with me doing the chemistry, and John
 working on the yeast biology. At Cornell, there was one laboratory that was
 the obvious place for such an experiment, and that was the lab of Ray Wu
 in the Department of Biochemistry. Ray was already well known for
 determining the sequence of the sticky ends of phage lambda, the first
 ever DNA to be sequenced, and his lab was deeply involved in the study of
 enzymes that could be used to manipulate and sequence DNA more
 effectively. John and I approached Professor Wu, who listened to our proposal
 and allowed that it was an interesting idea worth exploring. However
 he was reluctant to appear to be &lsquo;poaching' a graduate student from another
 lab and department; another complication was that the work would require
 collaboration with Fred Sherman's lab. John applied to Fred's lab in nearby
 Rochester, New York, for a postdoctoral position, and was accepted. At
 Cornell, I persisted and eventually Ray allowed me to transfer into his lab and
 begin the project.</p>
 <p>The interlude between wrapping up my work in the Department of
 Plant Physiology and starting as a transfer student in the Department of
 Biochemistry provided me with the opportunity for an extended vacation and
 my first trip to Europe on my own. I began with a visit to Cambridge, England
 where I was very kindly hosted by Professor Poole (for whom I had worked
 at McGill), who was on sabbatical at the University of Cambridge. I explored
 the town and was incredibly impressed by the Chapel of King's College and
 the ethereal music therein. Even more impressive was the famous Laboratory
 of Molecular Biology at the MRC, where I talked with one of the iconic figures
 of molecular biology, <a href="/nobel_prizes/medicine/laureates/2002/">Sydney Brenner</a>. I was asked to wait for Sydney in
 his office, which I was surprised to notice held two large desks, both piled to
 the ceiling with papers. When Sydney arrived he told me about his remarkable
 new project involving the use of the nematode <em>Caenorhabditis elegans </em>as a
 model system for developmental genetics &minus; this was an impressive if somewhat
 painful lesson on the right way to carry forward such an ambitious project. I also learned why two desks crammed that small office &minus; it turned out that
<a href="/nobel_prizes/medicine/laureates/1962/">Francis Crick</a>!</p>
 <p>After a memorable month of art, architecture and music in Paris, I
 returned to Ithaca to start afresh in a new lab with a new project. My goal
 was clear &minus; the chemical synthesis of the oligonucleotide needed for our
 gene detection scheme. At the time, this was still a challenging endeavor for
 a student such as myself with minimal synthetic skills. Ray had an ongoing
 collaboration with Saran Narang, who was developing the solution phase
 phosphotriester approach to oligonucleotide synthesis. Our plan was to use
 this approach to prepare large quantities of the five trimers needed to make
 a 15-mer, then link the trimers together to form 6-mers, a 9-mer and finally
 a 15-mer. I began the work under the tutelage of Chander Bahl, a postdoc
 who had some experience with this technology. Unfortunately our lab was
 better equipped for enzymology than synthesis, and we lacked a critical mass
 of experienced chemists. After a year of work, I was still far from my goal and
 becoming increasingly frustrated. Fortunately Ray Wu realized that I needed
 help, and arranged for me to visit Saran Narang's lab in Ottawa. There I was
 fortunate to receive training from Keichi Itakura, who later became famous
 for synthesizing the gene for insulin. After two weeks of intense training, I returned
 to Ithaca, and attacked my synthesis with fresh energy. A few months
 later, I was rewarded with several milligrams of our long sought 15-mer. In
 collaboration with John Stiles and Fred Sherman, who sent us RNA and DNA
 samples from appropriate yeast strains, we were able to show that we could
 use the labeled 15-mer as a probe to detect the <em>cyc1 </em>mRNA, and later the
 gene itself. This was quite exciting, and seeing our work published in <em>Nature</em>
 (2) was a great boost to my confidence after years of work with little to show.
 It was also an important lesson in effective research strategy, imprinting on
 me the value of seeking help from knowledgeable people when faced with
 difficulties. One of the delights of the world of science is that it is filled with
 people of good will who are more than happy to assist a student or colleague
 by teaching a technique or discussing a problem.</p>
 <p>The completion of my Ph.D. in 1977 marked the beginning of a
 major scientific transition for me. Against all commonsense advice, I
 decided to remain in Ray's lab for postdoctoral work, but in a very different
 scientific area. The decision was triggered by the arrival in Ray's lab of a new
 postdoc, Rodney Rothstein, from Fred Sherman's lab in Rochester. Rod was
 already a seasoned yeast geneticist, but had little experience with molecular
 biology; in contrast my graduate work was in molecular biology but I had no
 practical experience with genetics. We hit it off and essentially trained
 each other through our collaborative work on yeast transformation. Our
 frequent discussions were long and often loud, sometimes triggering mild
 protests from Ray who would emerge from his office and ask us to turn it
 down a notch when he needed a quieter atmosphere in which to work. The
 combination of the molecular biology I learned in Ray's lab and the
 genetics I learned there from Rod prepared me well for the next decade
 of my work on yeast, first in recombination studies, and later in telomere studies and other aspects of yeast biology. Ray was a wonderful advisor (3),
 and in addition to his scientific advice I absorbed much of his way of running
 a lab, which in essence was to be there when advice was needed but otherwise
 to let creative students and postdocs run with their ideas (Figure 2).</p>
 
<img src="szostak_autobio_2.jpg" alt="Ray Wu&rsquo;s lab, circa 1978">
<p class="photo-gallery-caption"><i>Figure 2</i>. 
Ray Wu's lab, circa 1978. Top left, my graduate and postdoctoral advisor Ray Wu;
 next to him is Rodney Rothstein, who introduced me to yeast genetics. I am seated at the
lower right.</p>
 

 <p>&nbsp;</p>
 
 <p>My postdoctoral studies of recombination in yeast were enabled by the
discovery, in Gerry Fink's lab at Cornell, of a way to introduce foreign DNA
into yeast (4). These pioneering studies of yeast transformation showed that
circular plasmid DNA molecules could on occasion become integrated into
yeast chromosomal DNA by homologous recombination. Rod and I began to
search for ways of increasing the frequency with which transformants were
recovered. Increasing the target size for recombination seemed like a good
possibility, and indeed when I transformed yeast with plasmids containing
fragments of rDNA, I did recover more transformants, and these contained
plasmid DNA integrated at the rDNA locus. These strains allowed me to
initiate studies of unequal sister chromatid exchange in rDNA locus,
resulting in my first publication in the field of recombination (5). Towards
the end of my stay in Ray Wu's lab, Rod and I came upon the first hints of
double-strand break stimulated recombination in yeast. Our preliminary
experiments suggested that cutting plasmid DNA within a region of homology
to yeast chromosomal DNA led to an increase in the recovery of transformants,
presumably reflecting increased recombination of the input DNA
with the homologous chromosomal locus. The idea that you could increase
transformation frequency by cutting the input DNA was pleasingly counterintuitive
and led us to continue our exploration of this phenomenon.</p>
 <p>My first independent position was at the Sidney Farber Cancer Institute
 (now the Dana-Farber Cancer Institute). I owe a great debt to Professor Ruth Sager, who was the main force behind hiring me. She established a terrific
 group of young investigators in her division, including Richard Kolodner
 and Gerry Rubin, creating a superb intellectual atmosphere. Ironically, I
 heard many years later that Ruth was only able to hire me over the objections
 of some of the senior clinical faculty, who did not believe that studies of yeast
 had any place in a cancer institute. Times have changed, and fortunately
 model systems are now much more widely appreciated. My graduate students
 came from the graduate program at Harvard Medical School, where I had
 an academic appointment in the Department of Biological Chemistry. These
 students were wonderful, and together we made rapid progress in setting up
 a productive yeast genetics lab.</p>
 <p>Our initial focus was the study of double-strand breaks in DNA and their
 repair by recombination. This work was spearheaded by my first graduate
 student, Terry Orr-Weaver, who is now a Professor at the Whitehead Institute
 and MIT. Terry's work, and our continuing interactions with Rod Rothstein,
 led us to think intensively about the kinds of reactions engaged in by
 DNA ends (6). There was considerable debate about different models for
 recombination within the wider DNA repair and recombination community,
 and seminars and conferences were important means for the exchange of
 the latest information. For many years, the major international recombination
 meeting was held in Aviemore, Scotland, which afforded the opportunity
 to sample diverse single-malts while discussing the intricacies of genetic
 exchange. I do recall that excessive sampling at one Aviemore meeting did
 make it difficult for me to present my work the next morning.</p>
 <p>I also enjoyed attending Gordon Conferences and Cold Spring Harbor
 meetings, which were small and highly interactive meetings that provided
 wonderful opportunities for young scientists to present their work and
 meet and talk to people doing the best and most important current work.
 In the summer of 1980, I attended the Nucleic Acids Gordon Conference,
 expecting to hear the latest advances in DNA synthesis, sequencing and
 repair. However, for me the high point of the meeting was hearing Liz
 Blackburn talk about her work on telomeres in <em>Tetrahymena</em>. Our subsequent
 discussion led to the initiation of a collaboration in which we decided to test
 the ability of <em>Tetrahymena </em>telomeres to function in yeast. Those experiments
 are described in my Nobel Lecture; here I will just say that it was an incredibly
 exciting time for me. I performed the experiments myself, and experienced
 the thrill of being the first to know that our wild idea had worked. It was
 clear from that point on that a door had been opened and that we were
 going to be able to learn a lot about telomere function from studies in yeast.
 Within a short time I was able to clone bona fide yeast telomeres, and in a
 continuation of the collaboration with Liz Blackburn's lab we soon obtained
 the critical sequence information that led us to propose the existence of the
 key enzyme, telomerase.</p>
 <p>With the success of the recombination and telomere projects, my lab
 began to grow. My second graduate student, Andrew Murray, now a Professor
 at Harvard, began to work on building artificial chromosomes. Andrew was a brilliant and energetic student who was fun to talk with about any
 conceivable experiment; his colorful personality (and dress) enlivened the
 lab. My collaboration with Rod and Terry grew to include Frank Stahl, the
 world's leading expert on the genetics of meiotic recombination, with whom
 we had many detailed discussions of the genetic implications of specific
 physical models. I particularly remember an afternoon I spent at Frank and
 Mary Stahl's house in Eugene, Oregon, going back and forth with Frank
 about different versions of the double-strand break repair model as we
 worked on our manuscript (7). It was an intense and stimulating experience
 that I still treasure.</p>
 <p>After five very productive years at the Farber, a remarkable opportunity
 induced me to move to the fledgling Department of Molecular Biology at
 the Massachusetts General Hospital (MGH). Howard Goodman, the
 founder of the Department and a major figure in the emerging field of biotechnology,
 had arranged an extremely interesting and innovative
 academia/industry collaborative venture. In this deal, the pharmaceutical
 giant Hoechst AG agreed to fully support all research in the MGH
 Department of Molecular Biology for a period of about ten years, in return
 for limited intellectual property rights. This was extremely attractive to
 me, as it promised to allow me to pursue research in any direction
 that I found to be of interest, without having to worry about obtaining
 traditional grant support for novel and hence untried ideas. Thus, in the
 summer of 1984 I moved my lab from the Farber to our new home in the
 downtown Boston campus of MGH (humorously referred to by colleagues at
 MIT's Whitehead Institute as &quot;one of the finest research institutes in downtown
 Boston&quot;).</p>
 <p>At that time, I was actively exploring the possibility of moving into other
 fields. By 1984, I had a growing feeling that my work in yeast was becoming
 less significant, in the sense that other people would inevitably end up doing
 the same experiments we were doing in a few months or years at the most. To
 learn more about other fields and to prepare myself to work in a new area I
 audited several courses at Harvard. A delightful course by Steve Kosslyn on
 cognitive psychology explored the fascinating correlations between localized
 brain lesions and cognitive deficits, and highlighted the emerging neuroimaging
 technologies that were promising to revolutionize studies of brain
 function. I also audited an applied math course to brush up on the skills I
 would need should I decide to seriously enter into structural biology. Finally
 an outstanding course on enzymology and catalytic mechanisms by the late
 Jeremy Knowles stimulated my interest in catalysis. Later, when Jeremy left
 science to become Dean of the Faculty of Arts and Sciences at Harvard, I had
 the good fortune to &quot;inherit&quot; one of his graduate students, Jon Lorsch, who
 migrated to my lab and did outstanding work on ribozyme selections and
 mechanistic enzymology.</p>
 <p>The combination of Jeremy's enzymology course and the recent discovery
 of ribozymes by <a href="/nobel_prizes/chemistry/laureates/1989/">Tom Cech and Sid Altman</a> (who shared the 1989 Nobel Prize
 in Chemistry for their work), ultimately led me to begin a transition to work on ribozymes. This seemed like a reasonably conservative way to switch fields,
 since the methods used to study ribozymes were largely a combination of
 molecular biology and chemistry. I was surprised that so few people were
 entering the field, since I thought that there were major questions to be
 addressed in terms of understanding the origins of biological catalysis in the
hypothetical RNA world that preceded the evolution of protein synthesis.</p>
 <p>I began to work with RNA myself, playing around with Cech's <em>Tetrahymena</em>
 ribozyme, which I obtained from the same piece of DNA that contained
 the <em>Tetrahymena </em>telomeres I had worked on just a few years earlier. The first
 student to join me in this new area was Jennifer Doudna. Jennifer had
 actually come to my lab to work on yeast genetics, but I was fortunate to
 persuade her that the future lay in RNA. Jennifer's energy and determination
 drove our efforts to convert self-splicing introns into an RNA replicase.
 We were soon joined by Rachel Green and several other dedicated students,
 techs, postdocs, and a memorable sabbatical visitor, Fran&ccedil;ois Michel, who
 impressed everyone with his work ethic, his uncanny ability to intuit structure
 from phylogeny, and his parallel career in butterfly evolution.</p>
<p>Even as I pushed our gradual transition to a focus on RNA, I maintained
 a substantial effort in yeast genetics for several years during the mid to late
 1980s. My interest in recombination and telomeres had not disappeared,
 and I wanted to bring our earlier advances to a satisfying conclusion.
 Recombination remained a large part of the lab, with Doug Treco, Alain
 Nicolas, Neil Schultes and Hong Sun maintaining a focus on the role of
 double-strand breaks in meiotic recombination. Most important for the
 telomere story was Vicki Lundblad's ground-breaking work on telomere
 genetics in yeast, which provided a link between telomere maintenance and
 senescence and aging (8). Barbara Dunn linked the telomere and recombination
 realms by study the transfer of sub-telomeric repeats between chromosomes
 by recombination.</p>
 
 <img src="szostak_autobio_3.jpg" alt="The Szostak lab, circa 1985">
<p class="photo-gallery-caption"><i>Figure 3</i>. The Szostak lab, circa 1985. Top row, from left: Neil Schultes, Andrew Murray, Dean Dawson, Neil Sugawara, Hong Sun. Bottom row, from left: Vicki Lundblad, Barbara Dunn, Jack Szostak, Stephanie Ruby.</p>

</table>
 <p>&nbsp;</p> 

<p>By end of the 80s, our yeast work was almost done, and the lab was
increasingly focused on RNA. The RNA floodgates really opened with the
work of Andy Ellington on <em>in vitro </em>selection (9), which ushered in a new era
of work on the <em>in vitro </em>directed evolution of new functional molecules. Over
time we came to feel that we could evolve a binding site for virtually any
target molecule, using any kind of nucleic acid. This confidence led us to
try to evolve new catalysts, and, returning to the RNA world hypothesis for
inspiration, we aimed for the chemistry of nucleic acid polymerization (10).
This was the basis of Dave Bartel's ground-breaking work on the selection of
ribozyme ligases, which he subsequently (in his own lab at the Whitehead
Institute and MIT) evolved into an RNA molecule with bona fide RNA
polymerase activity. Our advances fueled my interest in the role of RNA
in early evolution and seemed to bring the resurrection of the RNA world
almost within reach. Our ability to evolve new aptamers and ribozymes was
so intoxicating that my lab spent most of the 90s exploring the range of
possibilities and the limitations of what RNA could do. Our advances
began to attract attention, leading to my election to the National Academy
of Sciences and appointment as a Howard Hughes Investigator in 1998. At
the same time, the Hoechst funding of my department was winding down,
making my HHMI appointment particularly welcome as a means of enabling
ventures into new scientific areas.</p>
<p>As other labs also started to evolve new and interesting ribozymes, the
difficulty of evolving <em>de novo </em>proteins began to seem the greater challenge.
We entered the field of protein and peptide evolution when Richard W.
Roberts, a postdoc in my lab, learned how to trick the translation apparatus
into covalently linking a newly translated protein to its own mRNA through
the action of the antibiotic puromycin (11). Galvanized by this advance, I
encouraged several new lab members to develop and use this mRNA-display
technology to address fundamental questions about the origin of protein
structure. Most significantly, Tony Keefe used this method to evolve a novel
ATP-binding protein from a large library of random sequence polypeptides
(12). Remarkably, this non-biological protein looks indistinguishable from
any normal biologically derived small protein domain. Postdoctoral fellows
John Chaput and Sheref Mansy continued to evolve this protein and study its
structure over the following years.</p>
<p>The development of this protein evolution technology led me to co-found
a startup biotechnology company, together with Rich and my colleague
Brian Seed. Although the company was not a business success, it was a very
interesting and educational experience. The collaborative efforts of a team
of scientists ranging from protein biophysicists to people with clinical drug
development experience allowed us to evolve a small protein domain with
therapeutic potential; this artificially evolved protein is now in clinical
trials. While I have continued to maintain a focus on fundamental questions
in my laboratory, I firmly believe that small startup companies are the best
way to develop more applied research to the point that it can eventually be
therapeutically useful.</p>
<p>By the year 2000, I started to pay more attention to fundamental questions
  related to the origin of life. My interest in the role of compartmentalization
  and cellular structure in the origin of life was stimulated by discussions with
  Pier Luigi Luisi and David Bartel. A year of debate led to a synthesis of our
  views on the roles of genetics, compartmentalization and evolution, which
  we expressed in our 2001 <em>Nature </em>paper Synthesizing Life (13). This paper
  catalyzed my entry into the field of membrane biophysics, for I felt that
  having proposed a model for early cells in which bilayer membranes played a
  crucial role, it was incumbent on us to show that such models were physically
  plausible. I have to admit that I was somewhat surprised to find myself working
  with lipids and membranes, which are remarkably squishy and ill-defined
  by comparison with nucleic acids. However, in at least one way, the study
  of membranes composed of prebiotic building blocks such as fatty acids
  was perfect for me, since this field was filled with important yet technically
  addressable questions. When postdoctoral fellow Marty Hanczyc and graduate
  student Shelly Fujikawa joined the project, we were able to make rapid progress,
  and within a few years had demonstrated a proof-of-principle path for vesicle
  growth and division based solely on physical processes. I began to grow
  more confident that it might ultimately be possible to deduce plausible
  explanations for at least some of the mysterious steps in the origin of life. My
  enthusiasm grew when Irene Chen, a brilliant biophysics graduate student,
  made further progress by demonstrating a pathway for competition between
  protocells. We worried that our model protocells would not be able to take
  up nutrients, such as the nucleotides needed for the replication of their
  genetic material, but Sheref Mansy showed that this was not a problem.
  Most recently, another graduate student, Ting Zhu has come up with a very attractive pathway for spontaneous, coupled growth and division, so it is
  beginning to seem that the assembly and replication of protocell membranes
  is not as difficult as we once thought.</p>
  
  <img src="szostak_autobio_4.jpg" alt="Szostak lab group photo taken shortly after the Nobel Prize announcement">
<p class="photo-gallery-caption"><i>Figure 4</i>. Szostak lab group photo taken shortly after the Nobel Prize announcement.</p>


<p>&nbsp;</p> 

<p>The dramatic progress in the identification of pathways for the self-replication
of protocell membranes has encouraged us to focus on the hardest
remaining problem, the replication of the genetic material. Here the big
question is whether RNA was in fact the first genetic polymer, or whether
RNA was preceded by some simpler, easier to make or more robust genetic
material. This question has driven the most recent transformation of my
lab (Figure 4), into a well equipped synthetic organic chemistry lab. We are
synthesizing amino-nucleotides, the building blocks for phosphoramidate
polymers, due to their greater reactivity than normal nucleotides. Alonso
Ricardo, a postdoc, and Jason Schrum, a graduate student, have recently
made very significant progress in the template-directed synthesis of 2'-5'
linked phophoramidate DNA (14), and we are now exploring a series of
related polymers in a search for even better self-replicating genetic materials.
The complexity and fragility of RNA long made it seem an unlikely candidate
for the first genetic material, but this prospect has been revived by the
brilliant recent work from John Sutherland's lab in Manchester (15). With
John's former graduate student Matt Powner now in my lab as a postdoc, we
are eagerly exploring new avenues to the chemical replication of RNA. It is
thrilling to me to see people in my lab developing new approaches to the
synthesis of modified nucleic acids, but the suspense is almost unbearable as
we await the results of template-directed polymerization experiments.</p>
<p>From our current vantage point, it is not clear whether there will be many
solutions to the problem of chemically replicating genetic polymers, or
just one, or none, but in any case it is an exciting quest. Encouraged by our
small advances on the way, we are continuing to feel our way towards the
tantalizing goal of building replicating, evolving chemical systems.</p>
<p>&nbsp;</p>

<table cellpadding="2" summary="References">
  <tr>
<td class="smalltext"><b>References</b></td>
  </tr>
  <tr>
<td class="smalltext">1. Szostak, J.W., Sparkuhl, J., Goldstein M.E., &quot;Sexual induction in Eudorina: effects of
light, nutrients and conditioned medium,&quot; <em>J Phycol</em>. 1973; 9:215&ndash;218.</td>
  </tr>
  <tr>
<td class="smalltext">2. Szostak, J.W., Stiles, J.I., Bahl, C.P., Wu, R., &quot;Specific binding of a synthetic oligonucleotide to yeast cytochrome c mRNA,&quot; <em>Nature </em>1977; 265:61&ndash;63.</td>
  </tr>
  <tr>
<td class="smalltext">3. Szostak, J.W., &quot;Ray Wu, as remembered by a former student,&quot;. <em>Sci China C Life Sci.</em>2009; 52:108&ndash;110.</td>
  </tr>
  <tr>
<td class="smalltext">4. Hinnen, A., Hicks, J.B., Fink, G.R., &quot;Transformation of yeast,&quot; <em>Proc. Natl. Acad. Sci.USA </em>75, 1929&ndash;1933 (1978).</td>
  </tr>
  <tr>
<td class="smalltext">5. Szostak, J.W., Wu, R., &quot;Unequal crossing over in the ribosomal DNA of Saccharomyces cerevisiae,&quot; <em>Nature </em>1980; 284:426&ndash;430</td>
  </tr>
  <tr>
<td class="smalltext">6. Orr-Weaver, T.L., Szostak, J.W., Rothstein, R.J., &quot;Yeast transformation: a model system
for the study of recombination,&quot; <em>Proc. Natl. Acad. Sci. USA </em>1981; 78:6354&ndash;6358.</td>
  </tr>
  <tr>
<td class="smalltext">7. Szostak, J.W., Orr-Weaver, T.L., Rothstein R.J., Stahl, F., &quot;The double-strand-break
repair model for recombination,&quot; <em>Cell </em>1983; 33:25&ndash;35.</td>
  </tr>
  <tr>
<td class="smalltext">8. Lundblad, V. and Szostak, J.W., &quot;A mutant with a defect in telomere elongation leads
to senescence in yeast,&quot; Cell 57, 633&ndash;643 (1989).</td>
  </tr>
  <tr>
<td class="smalltext">9. Ellington, A.E. and Szostak J.W., &quot;In vitro selection of RNA molecules that bindspecific ligands,&quot; <em>Nature </em>346, 818&ndash;822 (1990).</td>
  </tr>
  <tr>
<td class="smalltext">10. Bartel, D.P. and Szostak, J.W., &quot;Isolation of new ribozymes from a large pool of randomsequences,&quot; <em>Science </em>261, 1411&ndash;1418 (1993).</td>
  </tr>
  <tr>
<td class="smalltext">11. Roberts, R.W. and Szostak, J.W., &quot;RNA-peptide fusions for the in vitro selection ofpeptides and proteins,&quot; <em>Proc. Natl. Acad. Sci. USA </em>94, 12297&ndash;12302 (1997).</td>
  </tr>
  <tr>
<td class="smalltext">12. Keefe, A.D. and Szostak, J.W., &quot;Functional proteins from a random sequence library,&quot;Nature 410, 715&ndash;718 (2001).</td>
  </tr>
  <tr>
<td class="smalltext">13. Szostak, J.W., Bartel, D.P., Luisi, P.L., &quot;Synthesizing life,&quot; <em>Nature, </em>2001; 409:387&ndash;390.</td>
  </tr>
  <tr>
<td class="smalltext">14. Schrum, J., Ricardo, A., Krishnamurthy, K., Blain, J.C. and Szostak, J.W., &quot;Efficient and rapid template-directed nucleic acid copying using 2'-amino-2', 3'-dideoxyribonucleoside-5'-phosphorimidazolide monomers,&quot; <em>J. Am. Chem. Soc. </em>31, 14560&ndash;14570 (2009).</td>
  </tr>
  <tr>
<td class="smalltext">15. Powner, M.W., Gerland, B., Sutherland, J.D. 2009, &quot;Synthesis of activated pyrimidineribonucleotides in prebiotically plausible conditions,&quot; <em>Nature </em>459:239.</td>
  </tr>
</table>
<p>&nbsp;</p>
